client/src/API/AuthMethods.ts

```diff
@@ -1,31 +1,67 @@
-import axios from "axios";
+import axios, { AxiosError } from "axios";
 import { LoginData, UserInfoData, URL } from "../constants";
+import { useHistory } from "react-router-dom";
+
+const history = useHistory();
 
 export const signUp = async (userInfo: UserInfoData) => {
   try {
-    await axios.post(`${URL}/auth/signUp`, {
-      email: userInfo.email,
-      password: userInfo.password,
-      first_name: userInfo.firstName,
-      last_name: userInfo.lastName,
-    });
-  } catch (e: any) {
-    // Will need to research more on how to better type the error
-    if (e.response.status > 399) {
-      alert(e.response.data.message[0]);
+    const response = await axios.post(
+      `${URL}/auth/signUp`,
+      {
+        email: userInfo.email,
+        password: userInfo.password,
+        first_name: userInfo.firstName,
+        last_name: userInfo.lastName,
+      },
+      {
+        withCredentials: true,
+        headers: {
+          Authorization: `Bearer ${sessionStorage.getItem("authToken")}`,
+        },
+      }
+    );
+    sessionStorage.setItem("authToken", response.data.accessToken);
+    if (response) {
+      history.push("/");
```

Comment:
I think it may be a good idea to call the logIn method right here.

The user signs up and is automatically logged in and sent to the home page?

So if the signup has a successful response, let's login the user and then send them on their way.

Also, it may be a good idea to show some sort of a successful account creation message when the signup works. So they know what to expect.